config.yaml

```diff
@@ -1,5 +1,6 @@
 common:
  log_file: "/var/log/host-os/builds.log"
+ log_size: 2097152
```

Comment:
I prefer to remove all the default values from the command line parser in a separate patch